connector/server/handlers/survival_analysis.go

```diff
@@ -0,0 +1,72 @@
+package handlers
+
+import (
+	survivalserver "github.com/ldsec/medco/connector/server/survivalanalysis"
+
+	"github.com/go-openapi/runtime/middleware"
+	"github.com/ldsec/medco/connector/restapi/models"
+	"github.com/ldsec/medco/connector/restapi/server/operations/survival_analysis"
+	"github.com/sirupsen/logrus"
+)
+
+// MedCoSurvivalAnalysisHandler handles /survival-analysis API endpoint
+func MedCoSurvivalAnalysisHandler(param survival_analysis.SurvivalAnalysisParams, principal *models.User) middleware.Responder {
+
+	survivalAnalysisQuery := survivalserver.NewQuery(
+		principal.ID,
+		param.Body.ID,
+		param.Body.UserPublicKey,
+		param.Body.CohortName,
+		param.Body.SubGroupDefinitions,
+		int(param.Body.TimeLimit),
+		param.Body.TimeGranularity,
+		param.Body.StartConcept,
+		param.Body.StartModifier,
+		param.Body.EndConcept,
+		param.Body.EndModifier,
+	)
+	logrus.Debug("survivalAnalysis: ", survivalAnalysisQuery)
+	err := survivalAnalysisQuery.Validate()
+	if err != nil {
+		logrus.Error(err)
+		return survival_analysis.NewSurvivalAnalysisDefault(400).WithPayload(
```

Comment:
Also here, it should use the proper generated object, something like `NewSurvivalAnalysisBadRequest`.